src/pages/transfer.tsx

```diff
@@ -93,10 +95,34 @@ const TransferPage = () => {
         setAssets(_assets);
       }
     } else {
-      const _assets = await AssetRegistry.getAssetsOnBlockchain(
-        RELAY_CHAIN,
-        chains[sourceChainId].paraId
+      const chainData = (await getChains()).find(
+        (chain) => chain.paraId ?
+          chain.paraId === sourceChainId && chain.relay.id === RELAY_CHAIN
+          :
+          sourceChainId === 0 && chain.id === RELAY_CHAIN
       );
+
+      const _assets = [];
+      if (chainData) {
+        const tokens = (await getTokens()).filter((token) => {
+          const isPartOfSourceChain = token.data.id.startsWith(chainData.id);
+          return isPartOfSourceChain;
+        });
+        const assets: Asset[] = tokens.map(t => {
+          const asset: Asset = {
+            asset: "",
+            name: t.data.symbol,
```

Comment:
`name: t.data.name`?

Reply:
Weirdly enough it does not have a `name` field.